crs_perception/include/crs_perception/model_to_point_cloud.hpp

```diff
@@ -0,0 +1,33 @@
+#include <vtkOBJReader.h>
+#include <vtkSmartPointer.h>
+#include <pcl/io/pcd_io.h>
+#include <pcl/point_types.h>
+
+namespace crs_perception
+{
+
+  class ModelToPointCloud
+  {
+  public:
+    ModelToPointCloud(std::string file_path, int num_samples, float leaf_size):
+      file_path_(file_path),
+      num_samples_(num_samples),
+      leaf_size_(leaf_size)
+    {
+    }
+
+    bool convertToPCL(pcl::PointCloud<pcl::PointXYZ>::Ptr &point_cloud);
```

Comment:
Generally, it's preferable to pass smart pointers by value, if the point cloud is an output argument you would then pass the Cloud object by reference.

Reply:
Noted